migrations/20230915230734_tap_ravs.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS scalar_tap_latest_ravs (
```

Comment:
Nah, not really. I picked a more explicit name (only stores the latest rav for any given (allocation, sender). But not necessary. I can shorten this.

Reply:
Well, also I made a mistake since it doesn't match this: https://github.com/graphprotocol/indexer-rs/blob/f7447f7f27a8436b74b4d20ce7abb878e576c354/migrations/20230915230734_tap_ravs.down.sql#L1 ...

Reply:
Fixed in 1ce92444b7287775bd4f2134bb08b06cdedce72b